Part_2/UIandFriends/AdminGUI.java

```diff
@@ -0,0 +1,1031 @@
+		tFlightIdNumberField.setFont(new Font("Tahoma", Font.PLAIN, 14));
+		tFlightIdNumberField.setEditable(false);
+		tFlightIdNumberField.setColumns(10);
+		tFlightIdNumberField.setBounds(214, 45, 86, 25);
+		panel.add(tFlightIdNumberField);
+		
+		JLabel lblSearchForTickets = new JLabel("Search for Tickets by:");
+		lblSearchForTickets.setFont(new Font("Tahoma", Font.BOLD, 24));
+		lblSearchForTickets.setBounds(10, 11, 310, 35);
+		browseTicketPanel.add(lblSearchForTickets);
+		
+		JButton btnDeleteSelectedTicket = new JButton("Delete Selected Ticket");
+		btnDeleteSelectedTicket.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				String message = adminBE.deleteTicket(tTicketIdNumberField.getText());
+				JOptionPane.showMessageDialog(frmAdminApplication.getComponent(0), message);
+			}
+		});
+		btnDeleteSelectedTicket.setBounds(404, 458, 220, 32);
+		browseTicketPanel.add(btnDeleteSelectedTicket);
+		btnDeleteSelectedTicket.setFont(new Font("Tahoma", Font.PLAIN, 16));
+		
+		JLabel label = new JLabel("Please Enter First Name");
+		label.setFont(new Font("Tahoma", Font.PLAIN, 20));
+		label.setBounds(20, 85, 276, 40);
+		bookTicketPanel.add(label);
+		
+		JLabel label_1 = new JLabel("Booking Ticket");
+		label_1.setFont(new Font("Tahoma", Font.BOLD, 24));
+		label_1.setBounds(20, 21, 287, 39);
+		bookTicketPanel.add(label_1);
+		
+		firstNameField = new JTextField();
+		firstNameField.setFont(new Font("Tahoma", Font.PLAIN, 16));
+		firstNameField.setColumns(10);
+		firstNameField.setBounds(20, 126, 265, 32);
+		bookTicketPanel.add(firstNameField);
+		
+		JLabel label_2 = new JLabel("Please Enter Last Name");
+		label_2.setFont(new Font("Tahoma", Font.PLAIN, 20));
+		label_2.setBounds(23, 169, 276, 40);
+		bookTicketPanel.add(label_2);
+		
+		lastNameField = new JTextField();
+		lastNameField.setFont(new Font("Tahoma", Font.PLAIN, 16));
+		lastNameField.setColumns(10);
+		lastNameField.setBounds(23, 207, 265, 32);
+		bookTicketPanel.add(lastNameField);
+		
+		JButton button_1 = new JButton("Back");
+		button_1.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				browseFlightPanel.setVisible(true);
+				bookTicketPanel.setVisible(false);
+			}
+		});
+		button_1.setFont(new Font("Tahoma", Font.PLAIN, 16));
+		button_1.setBounds(184, 452, 101, 32);
+		bookTicketPanel.add(button_1);
+		
+		JButton button_2 = new JButton("Confirm Ticket Purchase");
+		button_2.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				//String message = adminBE.bookTicket(bfirstNameField.getText(), blastNameField.getText(), bFlightIdNumberField.getText());
+				//JOptionPane.showMessageDialog(frmAdminApplication.getComponent(0), message);
+			}
+		});
+		button_2.setFont(new Font("Tahoma", Font.PLAIN, 16));
+		button_2.setBounds(317, 452, 254, 32);
+		bookTicketPanel.add(button_2);
+		
+		JButton button_3 = new JButton("Clear Name Fields");

```

Comment:
Since the only fields on this page are name fields, we probably can just say "Clear Fields"